development/install-prow.sh

```diff
@@ -24,6 +24,22 @@ if [ -z "${LOCATION}" ]; then
     LOCATION="global"
 fi
 
+if [ -z "$KUBECONFIG" ]; then
+      echo "\$KUBECONFIG is empty"
```

Comment:
How about using default config `readonly CONFIG="${HOME}/.kube/config"` in such case?

Reply:
done